.github/workflows/stale-issues.yaml

```
@@ -0,0 +1,38 @@
+name: Mark stale issues
+
+on:
+  schedule:
+    - cron: '30 1 * * *'  # Runs daily at 01:30 UTC
+  workflow_dispatch:
+
+permissions:
+  issues: write
```

To make ossf scorecard happier, let's set the permissions for the whole workflow to `issues: read` and only assign the write permissions to the job that needs it.

```suggestion
permissions:
  issues: read
```